realtime/redisSubscriber.js

```diff
@@ -21,14 +22,21 @@ const sub = require('../cache/redisCache').client.sub;
  */
 module.exports = (io) => {
   sub.on('message', (channel, mssgStr) => {
-    // message object to be sent to the clients
-    const mssgObj = JSON.parse(mssgStr);
-    const key = Object.keys(mssgObj)[0];
+    zlib.unzip(Buffer.from(mssgStr, 'base64'), (err, uncompressedBuffer) => {
```

Comment:
behind feature flag

we should try to identify here whether the message has been deflated or not--that way if we hit an error deflating it, we can send it raw, and not try to unzip it here